pandas/core/indexes/interval.py

```diff
@@ -251,14 +273,39 @@ def __new__(
     def from_breaks(
         cls,
         breaks,
-        closed: str = "right",
+        inclusive=None,
+        closed: lib.NoDefault = lib.no_default,
         name: Hashable = None,
         copy: bool = False,
         dtype: Dtype | None = None,
     ) -> IntervalIndex:
+
```

Comment:
i might be worth making a single helper function to use in all the places you added code just like this

Reply:
Hi @jreback Are you meaning to create new helper function for this code section 
```
if inclusive is not None and not isinstance(closed, lib.NoDefault):
    raise ValueError(
        "Deprecated argument `closed` cannot be passed "
        "if argument `inclusive` is not None"
    )
elif not isinstance(closed, lib.NoDefault):
    warnings.warn(
        "Argument `closed` is deprecated in favor of `inclusive`.",
        FutureWarning,
        stacklevel=2,
    )
    if closed is None:
        inclusive = "both"
    elif closed in ("both", "neither", "left", "right"):
        inclusive = closed
    else:
        raise ValueError(
            "Argument `closed` has to be either"
            "'both', 'neither', 'left' or 'right'"
        )
```
Right?

Reply:
yes exactly

Reply:
ok. Sure. Have add helper function in `pandas/_libs/interval.pyx`